xcube/api/gen/rbins/iproc.py

```diff
@@ -52,14 +52,50 @@ def get_reprojection_info(self, dataset: xr.Dataset) -> ReprojectionInfo:
                                 xy_crs=CRS_WKT_EPSG_4326,
                                 xy_gcp_step=1)
 
-    def get_time_range(self, dataset: xr.Dataset) -> Tuple[float, float]:
+    def get_time_range_v1(self, dataset: xr.Dataset) -> Tuple[float, float]:
```

Comment:
What is the use of `get_time_range_v1`? If it is no longer used, please remove it.

Reply:
yes, is not longer used, I will remove it